x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/EsqlPlugin.java

```diff
@@ -102,6 +104,42 @@ public class EsqlPlugin extends Plugin implements ActionPlugin {
         Setting.Property.Dynamic
     );
 
+    public static final Setting<TimeValue> ESQL_SLOWLOG_THRESHOLD_QUERY_WARN_SETTING = Setting.timeSetting(
+        "esql.slowlog.threshold.query.warn",
+        TimeValue.timeValueNanos(-1),
+        TimeValue.timeValueMillis(-1),
```

Comment:
Is there some reason for the default to be -1ns instead of -1ms? I guess both would work the same, right?

Reply:
I _think_ it's the same. let me change it